Misleading error message when project arguments points to folder that lacks 'tsconfig.json'

create new folder with name '1', run compiler passing path to this folder as a value of 'project' argument.
Error message: 
```
error TS5014: Failed to parse file '1/tsconfig.json': Unexpected token u.
```

--- a/src/compiler/tsc.ts
+++ b/src/compiler/tsc.ts
@@ -349,6 +349,11 @@ namespace ts {
 
             const result = parseConfigFileTextToJson(configFileName, cachedConfigFileText);
             const configObject = result.config;
+            if (!configObject) {
+                reportDiagnostics([result.error], /* compilerHost */ undefined);
+                sys.exit(ExitStatus.DiagnosticsPresent_OutputsSkipped);
+                return;
+            }
             const configParseResult = parseJsonConfigFileContent(configObject, sys, getDirectoryPath(configFileName));
             if (configParseResult.errors.length > 0) {
                 reportDiagnostics(configParseResult.errors, /* compilerHost */ undefined);

--- a/src/compiler/tsc.ts
+++ b/src/compiler/tsc.ts
@@ -346,6 +346,12 @@ namespace ts {
                     return;
                 }
             }
+            if (!cachedConfigFileText) {
+                const error = createCompilerDiagnostic(Diagnostics.File_0_not_found, configFileName);
+                reportDiagnostics([error], /* compilerHost */ undefined);
+                sys.exit(ExitStatus.DiagnosticsPresent_OutputsSkipped);
+                return;
+            }
 
             const result = parseConfigFileTextToJson(configFileName, cachedConfigFileText);
             const configObject = result.config;

